<launch>
<node pkg="arena_timespace_planner" name="plan_manager" type="time_space_planner_node" output="screen">
    <!-- plan manager -->
    <rosparam file="$(find arena_timespace_planner)/launch/config/plan_manager_param.yaml" command="load" ns="plan_manager"/> <!-- ns="global_costmap" -->
    
    <!-- Mapping -->
    <rosparam file="$(find arena_timespace_planner)/launch/config/gridmap_param.yaml" command="load" ns="sdf_map"/> <!-- ns="global_costmap" -->
    
    <!-- global planner -->
    <rosparam file="$(find arena_timespace_planner)/launch/config/global_plan_optimization_param.yaml" command="load" ns="optimization_ESDF"/> 
    <rosparam file="$(find arena_timespace_planner)/launch/config/global_plan_param.yaml" command="load" ns="kino_astar"/> 
    
    <!-- mid planner -->
    <rosparam file="$(find arena_timespace_planner)/launch/config/timed_astar_param.yaml" command="load" ns="timed_astar"/> <!-- ns="global_costmap" -->
    <rosparam file="$(find arena_timespace_planner)/launch/config/local_plan_optimization_param.yaml" command="load" ns="optimization"/> <!-- ns="global_costmap" -->

</node>
</launch>